truss/remote/remote_cli.py

```diff
@@ -15,6 +25,7 @@ def inquire_remote_config() -> RemoteConfig:
     api_key = inquirer.secret(
         message="🤫 Quietly paste your API_KEY:",
         qmark="",
+        validate=NonEmptyValidator(),
```

Comment:
I wonder if we should make a more sophisticated validator in a followup to ensure that it's a valid API key. I'll make a note to do that as a followup.

Reply:
yeah, I think that's a good idea. will merge this for now, but we can adapt this validator